<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StudyImage</class>
 <widget class="QMainWindow" name="StudyImage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>508</width>
    <height>366</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QDateEdit" name="m_startDate">
       <property name="wrapping">
        <bool>false</bool>
       </property>
       <property name="readOnly">
        <bool>false</bool>
       </property>
       <property name="dateTime">
        <datetime>
         <hour>0</hour>
         <minute>0</minute>
         <second>0</second>
         <year>1969</year>
         <month>2</month>
         <day>1</day>
        </datetime>
       </property>
       <property name="currentSection">
        <enum>QDateTimeEdit::YearSection</enum>
       </property>
       <property name="displayFormat">
        <string>yyyyMMdd</string>
       </property>
       <property name="calendarPopup">
        <bool>true</bool>
       </property>
       <property name="timeSpec">
        <enum>Qt::LocalTime</enum>
       </property>
       <property name="date">
        <date>
         <year>1969</year>
         <month>2</month>
         <day>1</day>
        </date>
       </property>
      </widget>
      <widget class="QLabel" name="m_To">
       <property name="text">
        <string>  è‡³  </string>
       </property>
      </widget>
      <widget class="QDateEdit" name="m_endDate">
       <property name="wrapping">
        <bool>false</bool>
       </property>
       <property name="readOnly">
        <bool>false</bool>
       </property>
       <property name="dateTime">
        <datetime>
         <hour>0</hour>
         <minute>0</minute>
         <second>0</second>
         <year>2021</year>
         <month>2</month>
         <day>1</day>
        </datetime>
       </property>
       <property name="currentSection">
        <enum>QDateTimeEdit::YearSection</enum>
       </property>
       <property name="displayFormat">
        <string>yyyyMMdd</string>
       </property>
       <property name="calendarPopup">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QComboBox" name="m_StudyModality">
       <item>
        <property name="text">
         <string>ALL</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>CT</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>MR</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>CR</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>DR</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>RF</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>ES</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>US</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Other</string>
        </property>
       </item>
      </widget>
      <widget class="QPushButton" name="m_getStudyDbImages">
       <property name="text">
        <string>Check Study</string>
       </property>
       <property name="icon">
        <iconset resource="res.qrc">
         <normaloff>:/image/image/serch.png</normaloff>:/image/image/serch.png</iconset>
       </property>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QTableWidget" name="m_tableWidget">
      <property name="autoFillBackground">
       <bool>true</bool>
      </property>
      <property name="sortingEnabled">
       <bool>true</bool>
      </property>
      <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
       <bool>true</bool>
      </attribute>
      <attribute name="horizontalHeaderStretchLastSection">
       <bool>true</bool>
      </attribute>
      <attribute name="verticalHeaderStretchLastSection">
       <bool>true</bool>
      </attribute>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>508</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
